samples/templates/default-azuredeploy.json

```diff
@@ -245,8 +254,9 @@
             }
         },
         {
+            "condition": "[equals(parameters('solutionType'), 'FhirServerCosmosDB')]",
```

Comment:
We might want to split the Cosmos DB and SQL server ARM template out to separate file and include that conditionally? Doesn't have to be in this PR but it might be easier to read if we separate them by components?

Reply:
I think long term that will probably make sense. There's going to likely be a number of more options in the SQL arm template so I'm not sure currently if it makes more sense to make two completely separate templates or use a base template with two sub templates depending on the data store selected.

Reply:
Splitting out sql from cosmos arm templates.